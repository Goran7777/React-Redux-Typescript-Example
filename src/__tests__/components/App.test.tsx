// import * as React from 'react';
// import { shallow } from 'enzyme';
// import { App } from '../../components/App';

// describe('ExampleComponent', () => {
//   it('renders', () => {
//     const wrapper = shallow(<App />);
//     expect(wrapper).toMatchSnapshot();
//   });
// });
