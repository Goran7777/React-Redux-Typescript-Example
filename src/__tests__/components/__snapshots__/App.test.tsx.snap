// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExampleComponent renders 1`] = `
<div>
  <h1>
    Hello from App
  </h1>
</div>
`;
